GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -758,6 +767,26 @@ protected void invokeUrl(String serverBase, String urlString,
         return errorMap;
     }
 
+    /**
+     * When a blank or null cell is incurred while processing, make sure that this is the last cell
+     * in the row and ignore any other "phantom cells" that may occur. We only allow this for the
+     * header row. If the blank cell occurs in between valid header cells we return true.
+     *
+     * @param cell
+     * @return
+     */
+    private boolean isMissingHeaderCell(Cell cell) {
+        assert cell.getRow().getRowNum() == 0; // only process header rows
```

Comment:
I tend to be wary of assertions in non-tests, production code. Why would we want an AssertionError being thrown here? As far as I can tell, we are not handling that particular error. Also, are assertions enabled in GAE?